tests/test_actions.py

```diff
@@ -640,3 +654,181 @@ def test_describe(self) -> None:
         assert SeeAnyOf().describe() == "See if no tests pass 🤔."
         assert SeeAnyOf(test).describe() == "See if 1 test passes."
         assert SeeAnyOf(*tests).describe() == f"See if any of {len(tests)} tests pass."
+
+
+class SimpleQuestion(Answerable):
+    @beat("{} examines SimpleQuestion")
+    def answered_by(self, actor: Actor):
+        return True
+
+    def describe(self):
+        return "SimpleQuestion"
+
+
+class TestQuietly:
+    def test_can_be_instantiated(self) -> None:
+        q1 = Quietly(None)
+        q2 = Quietly(1)
+        q3 = Quietly(FakeQuestion())
+        q4 = Quietly(FakeAction())
+        q5 = Quietly(FakeResolution())
```

Comment:
From a philosophical standpoint, i think making sure the function returns the correct `Quietly*` class is a separate test function than this one.

I'm thinking:
 * 1 test that each of the `Quietly*` classes can be instantiated.
 * 1 test that each of the `Quietly*` classes implent the right protocol.
 * 1 test that the `Quietly` function produces the correct `Quietly*` class, depending on its argument.